controllers/goharbor/harborcluster/database/postgresql.go

```diff
@@ -1,15 +1,74 @@
 package database
```

Comment:
lcm controlles please move to dir `/pkg/cluster`

Reply:
copy